### PR TITLE
feat: 28 Files headers compatibility

### PR DESCRIPTION
**Pull request by skjnldsv**
https://github.com/nextcloud/server/pull/39808

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -22,67 +22,49 @@
...
-		})
-
-		if (fileList.getCurrentDirectory() === '/') {
-			vm.show()
-		}
-
-		return this.el
 	},
-
+	updated(folder, view) {},
```
Comment (skjnldsv) [inline src/main.js:67]: Maybe this is needed? I don't know. I think the enabled works already as there is technically only one folder which works there: the user root folder?

Comment (juliushaertl) [reply src/main.js:67]: Yep, seems not necessary for the recommendations as they are also only fetched once from the server and not updated on the fly

**Review by juliushaertl — APPROVED** [review]
Tested and works with the matching server branch 👍

**Comment by skjnldsv** [comment]
Seems good now! :rocket:

**Pull request merged**